### PR TITLE
stop depending directly on tensorflow

### PR DESCRIPTION
**Pull request by rom1504**
allows having a single tf yarn package for both cpu and gpu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,18 @@ Two alternatives to TensorFlow's distribution strategies are available:
 
 ### Install with Pip
 
+Note that in order to support both tensorflow cpu and gpu in a single package, tf-yarn doesn't depend directly on tensorflow.
```
Comment (nateagr) [inline README.md:23]: Could you please indicate that we recommend TensorFlow >=1.15.0,<=2.2.0 ?  We did not run test with newer versions and it won't work with older versions

File: README.md [hunk 2]
```diff
@@ -20,15 +20,18 @@ Two alternatives to TensorFlow's distribution strategies are available:
...
-$ pip install tf-yarn
+$ pip install tf-yarn[cpu]
 ```
 
 ### Install from source
 
 ```bash
 $ git clone https://github.com/criteo/tf-yarn
-$ cd tf-yarn
+$ cd tf-yarn[cpu]
```
Comment (nateagr) [inline README.md:34]: We don't have any directory tf-yarn[cpu]

File: README.md (hunk 1 above)
Comment (nateagr) [inline README.md:23]: Could you please indicate that we recommend TensorFlow >=1.15.0,<=2.2.0 ? We did not run test with newer versions and it won't work with older versions

Comment (rom1504) [reply README.md:23]: yes, done

File: README.md (hunk 2 above)
Comment (rom1504) [reply README.md]: indeed, typo, fixed

**Pull request merged**